src/main/java/org/agbrothers/englishbot/processing/processor/lesson/english/EnglishAudioLessonProcessor.java

```diff
@@ -0,0 +1,70 @@
+package org.agbrothers.englishbot.processing.processor.lesson.english;
+
+import org.agbrothers.englishbot.constant.State;
+import org.agbrothers.englishbot.entity.Lesson;
+import org.agbrothers.englishbot.entity.ResponseMessage;
+import org.agbrothers.englishbot.entity.Word;
+import org.agbrothers.englishbot.processing.ProcessingExchange;
+import org.agbrothers.englishbot.service.texttospeech.TextToSpeechClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.meta.api.objects.InputFile;
+
+import static org.agbrothers.englishbot.constant.CommonPhrase.MAKE_CHOICE_AUDIO;
+
+@Component
+public class EnglishAudioLessonProcessor extends AbstractEnglishLessonProcessor {
+
+    @Autowired
```

Comment:
@Autowired on fields is a bad practice. It uses reflection to inject beans and has some other drawbacks. Please move it to setter

Reply:
Ok. I created a setter with needed annotation